pkg/kubelet/cm/topologymanager/policy_options.go

```diff
@@ -71,11 +75,23 @@ func NewPolicyOptions(policyOptions map[string]string) (PolicyOptions, error) {
 				return opts, fmt.Errorf("bad value for option %q: %w", name, err)
 			}
 			opts.PreferClosestNUMA = optValue
+		case MaxAllowableNUMANodes:
+			optValue, err := strconv.Atoi(value)
+			if err != nil {
+				return opts, fmt.Errorf("bad value for option %q: %w", name, err)
+			}
+			opts.MaxAllowableNUMANodes = optValue
 		default:
 			// this should never be reached, we already detect unknown options,
 			// but we keep it as further safety.
 			return opts, fmt.Errorf("unsupported topologymanager option: %q (%s)", name, value)
 		}
 	}
+
+	// if user doesn't specify a new value, make defaultMaxAllowableNUMANodes
+	// is the recommended default.
+	if opts.MaxAllowableNUMANodes == 0 {
+		opts.MaxAllowableNUMANodes = defaultMaxAllowableNUMANodes
+	}
```

Comment:
let's move this initialization at the beginning, around line 65:
```
opts := PolicyOptions{
    MaxAllowableNUMANodes: defaultMaxAllowableNUMANodes,
}
```

Reply:
Hmm, I had left the same comment but it is lost now somehow...